### PR TITLE
Enhance tracking api

### PR DESCRIPTION
**Pull request by atrifan**
- unify log format
- enhance log_format

### CODE REVIEW

**Review by andralungu — APPROVED** [review]
LGTM 👍

**Review by vdatcu — APPROVED** [review]

File: api-gateway-config/conf.d/api_gateway_logging.conf [hunk 1]
```diff
@@ -29,4 +29,6 @@ log_format  platform  '$remote_addr - $remote_user [$time_local] request="$reque
                       'status=$status bbs=$body_bytes_sent rl=$request_length rt=$request_time hr="$http_referer" '
                       'ua="$http_user_agent" xfwdf="$http_x_forwarded_for" '
                       'upadd="$upstream_addr" upstat=$upstream_status uprt="$upstream_response_time" '
-                      'sid="$service_id" sname="$service_name" reqid=$requestId';
\ No newline at end of file
+                      'sid="$service_id" sname="$service_name" reqid=$requestId';
+
+log_format tracking '$remote_addr [$time_local] rt=$request_time trid=$tracking_id';
```
Comment (corinapurcarea) [inline api-gateway-config/conf.d/api_gateway_logging.conf]: change both of them to track_id

**Review by corinapurcarea — CHANGES_REQUESTED** [review]

File: api-gateway-config/conf.d/api_gateway_logging.conf (hunk 1 above)
Comment (atrifan) [reply api-gateway-config/conf.d/api_gateway_logging.conf]: check again

**Review by corinapurcarea — APPROVED** [review]
👍

**Review by vdatcu — APPROVED** [review]

**Pull request merged**